index.html

```diff
@@ -0,0 +1,459 @@
+          </p>
+          <ul class="cats">
+            <li>html</li>
+            <li>css</li>
+            <li>javasript</li>
+          </ul>
+          <button class="btn">See Projects</button>
+        </div>
+      </div>
+      <div class="card-works">
+        <img
+          src="./assets/img/Snapshoot Portfolio2.png"
+        />
+        <div class="card-details">
+          <h2 class="name">Multi-Post Stories</h2>
+          <div class="frame">
+            <ul class="frame-item">
+              <li>Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="p-detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="cats">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Projects</button>
+        </div>
+      </div>
+      <div class="card-works">
+        <img
+          src="./assets/img/Snapshoot Portfolio3.png"
+        />
+        <div class="card-details">
+          <h2 class="name">Tonic</h2>
+          <div class="frame">
+            <ul class="frame-item">
+              <li>Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="p-detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="cats">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Projects</button>
+        </div>
+      </div>
+      <div class="card-works">
+        <img
+          src="./assets/img/Snapshoot Portfolio2.png"
+        />
+        <div class="card-details">
+          <h2 class="name">Multi-Post Stories</h2>
+          <div class="frame">
+            <ul class="frame-item">
+              <li>Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="p-detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Projects</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <h2 class="name">About Myself</h2>
+      <p class="p-detail">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <p class="connect">Let's Connect</p>
```

Comment:
Kindly replace the `p` tag with an `hx` tag on your Let's Connect text. :+1: :smiley: :smiley: